src/components/Sidebar/SectionSelection.js

```diff
@@ -48,6 +49,7 @@ function SectionSelection({
             key={section.id}
             button
             onClick={() => addSection(section)}
+            disabled={!!scheduledSections && scheduledSections.some(s => s.id === section.id)}
```

Comment:
Let's try to avoid single letter variable names - I think in this case `scheduledSection` would be a good name instead of `s`. Also, just to keep the code cleaner, let's break all of this into a separate variable named "sectionAlreadyAdded" or something like that. You'll have to turn the implicit return arrow function into an arrow function with a function body (curly braces) and declare your variable inside that, but I think that'll still be cleaner.